js/hatrac.js

```diff
@@ -298,6 +298,7 @@ var ERMrest = (function(module) {
         this.CHUNK_QUEUE_SIZE = otherInfo.chunkQueueSize || 4;
 
         this.file = file;
+        this.storedFilename = file.name; // the name that will be used for content-disposition and filename column
```

Comment:
It seems like chaise might call this function without providing any `file` object. So you should check that file is an object before using `file.name`.

I found this while working on migrating test cases. In this specific tests, we're not providing any files for an asset input and just submitting. Chaise throws an error on this line.